src/send_push/index.ts

```diff
@@ -0,0 +1,90 @@
+import https from 'node:https';
+import { Command, Option } from 'commander';
+import { createLogger } from './logger';
+
+export const sendPushCommand = new Command('send-push')
+  .description('Send a rich push notification to a specified device.')
+  .requiredOption(
+    '--api-key <apiKey>',
+    'App API key for your customer.io workspace. You can visit https://fly.customer.io/settings/api_credentials?keyType=app to get your API key.'
+  )
+  .requiredOption('--token <token>', 'Device token to send push to')
+  .addOption(
+    new Option('--platform <platform>', 'Platform of the device')
+      .choices(['ios', 'android'])
+      .makeOptionMandatory()
+  )
+  .addOption(
+    new Option(
+      '--provider <provider>',
+      'Push provider to use to deliver the notification'
+    ).choices(['apns', 'fcm'])
+  )
+  .option(
+    '--deep-link <deepLink>',
+    'Deep link to send in the test push notification'
+  )
+  .action(send);
+
+type sendOptions = {
+  apiKey: string;
+  token: string;
+  platform: string;
+  provider: string;
+  deepLink: string;
+};
+
+async function send(options: sendOptions) {
+  const logger = createLogger();
+
+  const data = JSON.stringify({
+    device_token: options.token,
+    device_platform: options.platform,
+    deep_link: options.deepLink,
+    push_provider: options.provider,
```

Comment:
[It looks like `--provider` is optional without a default value](https://github.com/customerio/cio-sdk-tools/pull/23/files#diff-f25bd40e301d4faac091dfb2907db3c354fa8fa5bd0c15bea499f5eeacafac7eR17-R21). Therefore, this JSON field in the request payload could be null. Would that cause an issue with the request?

Reply:
the backend handles the request correctly and sets a default value if the key is missing from the payload